django/urls/resolvers.py

```diff
@@ -541,6 +545,7 @@ def resolve(self, path):
                             sub_match.url_name,
                             [self.app_name] + sub_match.app_names,
                             [self.namespace] + sub_match.namespaces,
+                            str(pattern),
```

Comment:
hmm, I think we should also be recursing on the `sub_match.route` here :thinking:

Reply:
This test with inclusion does not seem to pass: https://github.com/django/django/commit/4830fd0ed69e7a57acfdce5231bbc2798d949a01